crates/bitwarden-crypto/src/keys/symmetric_crypto_key.rs

```diff
@@ -53,6 +55,13 @@ impl SymmetricCryptoKey {
         SymmetricCryptoKey::Aes256CbcHmacKey(Aes256CbcHmacKey { enc_key, mac_key })
     }
 
+    /// Generate a new random [SymmetricCryptoKey] for unit tests. Note: DO NOT USE THIS
+    /// IN PRODUCTION CODE.
```

Comment:
If this is only available in bitwarden-crypto you can use `#[test]` to put it behind a compile flag.

Reply:
The idea for this function was to have something that teams can use to make their uses of generate determistic for tests, outside of the cryptoCrate. For instance  

```
#[test]
fn test_move_user_cipher_to_org_manually() {
    let org = uuid::Uuid::new_v4();
    let key = SymmetricCryptoKey::generate(rand::thread_rng());
    let org_key = SymmetricCryptoKey::generate(rand::thread_rng());
    let key_store = create_test_crypto_with_user_and_org_key(key, org, org_key);

```

Reply:
This was intended to be used outside of the crate too, in case teams need a deterministically generated key.

However, looking at this more widely, I'm not sure how this would actually help since the encryption is still non-deterministic (based on IV/nonce)...

Happy to mark it `#[cfg(test)]` for now and revisit later when we find cases where it would be useful?